core/trino-main/src/main/java/io/trino/execution/CreateFunctionTask.java

```diff
@@ -122,8 +121,9 @@ public ListenableFuture<Void> execute(CreateFunction statement, QueryStateMachin
         return immediateVoidFuture();
     }
 
-    private void verifyFormattedFunction(String sql, FunctionSpecification function)
+    private String formattedFunction(FunctionSpecification function)
```

Comment:
This also verifies. `verifiedFormattedfunction`?

Reply:
The verification is an internal check for the function implementation. It's not verifying something on the caller's behalf, so signaling it to the caller (or reader of the calling code) doesn't seem useful.